src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -203,8 +204,8 @@ public Token call() throws SaltStackException {
      * @return object representing the scheduled job
      * @throws SaltStackException if anything goes wrong
      */
```

Comment:
Comment is out-of date: this method does not return the job id any more. Same for the asynchronous version.